### DISCUSSION TITLE
怎样设置OpenClash才能排除掉访客网络呢？

### DISCUSSION THREAD

**Discussion by InspoOnU** [vernesong/OpenClash#3049] [Q&A]
路由器上设置了三个防火墙区域：WAN, LAN, GUEST。因为是双网口的软路由所以GUEST是虚拟VLAN隔离的。
如果只想让LAN区域里的设备走代理，有什么办法可以实现吗？

**Comment by vernesong** [comment]
访问控制白名单直接写你lan的cidr不就行了

**Reply by InspoOnU** [reply]
IPv6的CIDR用不了这个方法。IPv6的前缀会变，OpenWRT可以设置每个接口的前缀分配提示。比如LAN的前缀是xxxx:xxxx:xxxx:xxx2::/64，但是OpenClash好像不支持::2:0:0:0:0/::f:ffff:ffff:ffff:ffff这种写法。除了关掉IPv6代理还有别的办法吗？

**Reply by vernesong** [reply]
![image](https://user-images.githubusercontent.com/42875168/221923634-9f17b1c7-f6c3-4b76-bd3e-672720018543.png)
只能这种
